### DISCUSSION TITLE
Restore from Backup

### DISCUSSION THREAD

**Discussion by scarymachines** [snipe/snipe-it#11920] [Q&A]
Hello community,

after a hardware crash I need to restore my snipeit from a backup file. What is the best way to proceed now? Install the latest version and import the backup? And run the migrate script? Or would it also be possible to migrate to Docker in this move? Is it possible to migrate from a "normal" installation to Docker?

greetings

**Comment by follet-git** [comment]
As for me: install the new version and make import.
